### PR TITLE
Alert-Button

### PR DESCRIPTION
**Pull request by jatinmore**
Alert and Buttons Component added
 -fixed some css
 -fixed path for diff pages
 -screenshots
127 0 0 1_5502_components_Alert-Button_alert html(Surface Pro 7)
127 0 0 1_5502_components_Alert-Button_button html(Surface Pro 7)

### CODE REVIEW

**Review by bmendhe23 — APPROVED** [review]
Looks good to me.

File: styles/avatar.css [hunk 1]
```diff
@@ -38,6 +41,9 @@
 .description
 {
     font-size: larger;
+    margin-bottom: 1rem;
+
```
Comment (bhavyaj12) [inline styles/avatar.css]: Remove extra blanks

Comment (bhavyaj12) [reply styles/avatar.css]: And try not to use larger, instead you can define font size utility classes

**Review by bhavyaj12 — APPROVED** [review]
I think for the headings you could use a cleaner font, and for alert content as well. Rest code is good

**Comment by pavan6115** [comment]
Add some spacing between the heading and the subheading, rest is good

**Pull request merged**